README.md

```diff
@@ -14,17 +14,29 @@
 PyTorch lets you send a model or tensor `.to(device)`, so
 why can't you do `my_fn.to('a_gcp_a100')` or `my_table.to('parquet_in_s3')`?
 Runhouse allows just that: send code and data to any of your compute or
-data infra (with your own cloud creds), all in Python, and continue to use them
-eagerly exactly as they were.
+data infra (with your own cloud creds), all in Python, and continue to interact with
+them normally from your local environment.
+
+**Runhouse is not an orchestrator.** Think of Runhouse as an expansion pack to your Python
```

Comment:
if this could be simplified in the future or moved down, it's a lot of text at the beginning of the page

Reply:
I agree. It feels like we need to add it but I'd like to simplify and spread it around below shortly.